src/components/general/Footer.js

```diff
@@ -1,22 +1,90 @@
 import React from "react";
+import Link from "next/link";
 import { Box } from "@material-ui/core";
 import { makeStyles } from "@material-ui/core/styles";
 
 const useStyles = makeStyles(theme => ({
-  root: {
+  footer: {
```

Comment:
Why did this name change?

Reply:
I built a wrapper around it, removed it again and forgot to change it back. Reverted.